docs/en/observability/synthetics-manage-monitors.asciidoc

```diff
@@ -1,17 +1,11 @@
 [[synthetics-manage-monitors]]
-= Manage synthetic monitors
+= Manage browser monitors
 
 beta[]
 
 After you've <<synthetic-run-tests, created a synthetic monitor>>,
 you can update the contents of the tests that they run, update the monitor's configuration,
-and permanently delete monitors.
-
-The way you manage browser monitors will depend on how you created the monitor:
-
-* If you used <<synthetic-monitor-choose-project>>, you'll manage the monitors using the `monitor` API or in {kib}'s Monitor Management UI.
-* If you used <<synthetic-monitor-choose-heartbeat>>, you'll manage the monitors in the {heartbeat} config file.
-* If you used <<synthetic-monitor-choose-agent>>, you'll manage the monitors in {kib}'s Integrations UI.
+and permanently delete monitors using the `monitor` API or in {kib}'s Monitor Management UI.
```

Comment:
I wonder if this would better to say `push` instead of
> the `monitor` API

As it suggests there's a `delete` API endpoint (there will be an API in the future, but not yet that we're documenting)